### ISSUE TITLE
DL+ window position is not stored

### ISSUE THREAD

**Issue by andimik** [Opendigitalradio/dablin#73]
When you move the DL+ window and change the service, it's position will be on the top left again. Seems every appearance the position is set to default.

**Comment by basicmaster** [comment]
Hmm, I cannot reproduce this. When I'm on a service, move the DL+ window and change to another service of the same ensemble, the DL+ first hides. After the first DL+ information is received, it is visible again...at the very same position it was hidden some seconds ago.

Does this happen with the current DABlin version? On which operating system did you observe this?

**Comment by andimik** [comment]
Tested with 9f8426a8eb85e8aac9b19b9984b3eeb39fbacfd6 under Ubuntu

> at the very same position it was hidden some seconds ago.

No, not on my machine.

[Bildschirmaufzeichnung vom 26.02.2024, 22:25:35.webm](https://github.com/Opendigitalradio/dablin/assets/24510556/b3693712-98ad-49f8-87d2-618552dbbbfb)

**Comment by andimik** [comment]
Ubuntu 22.04.2 LTS
64 bit
GNOME version 42.5
Window manager Wayland

**Comment by andimik** [comment]
I checked on one of my Raspberry PI, and there it works ... Strange.

**Comment by andimik** [comment]
I have now the same problem under another Raspberry PI 4B+, the DL+ position is not stored.

Tested the master branch, 1.16.0 dated 7th April 2024.

**Comment by andimik** [comment]
I've started with a Live Image with Ubuntu 25.04 and only installed `dablin` via `apt-get` but I still have this behaviour.

So it has nothing to do with my local installation.

https://github.com/user-attachments/assets/09c311e7-3538-4e04-a986-b04351553530

**Comment by jacob-hofman** [comment]
Hi @andimik , as I wrote in #82 the problem seems to be caused by Wayland behaving differently than X in sending GTK events. The child windows for slideshow and DL+ should also appear at main window side, and not over the main one, and they should also move when you move the main window, but I can see in your last video that you're having the same behaviour I experience on Wayland. Switching to an X session will probably solve your issue for now.

**Comment by andimik** [comment]
Indeed, it's easy to switch to X11 at the login (icon bottom right).

https://github.com/user-attachments/assets/a0e34c11-5d56-4e24-a7bd-acbb8cbc12c1

I guess I can close this issue now.
